sites/public/layouts/application.tsx

```diff
@@ -34,7 +34,7 @@ const Layout = (props) => {
         <SiteHeader
           skip={t("nav.skip")}
           logoSrc="/images/detroit-logo.png"
-          notice={<div dangerouslySetInnerHTML={getTranslationMarkup("nav.getFeedback")}></div>}
+          notice=""
```

Comment:
Can we remove `notice=""` entirely?

Reply:
Unfortunately notice is a required parameter and without changing the UI-Component  we have to include something.